### PR TITLE
Adding new functionalities and refactoring JSP

### PR DESCRIPTION
**Pull request by dufajlusi**
Added new functionality 
- add user to team
- remove user from team

Refactoring JSP by separating into fragments...

### CODE REVIEW

File: src/main/java/eu/execom/labs/teamodoro/service/UserService.java [hunk 1]
```diff
@@ -63,4 +62,20 @@ public void addTeam(String googleId, Team team) {
         userTeamRepository.save(new UserTeam(user, team));
     }
 
+    @Transactional
+    public List<User> findAllUsers() {
+        return userRepository.findAll();
```
Comment (stanislavgrujic) [inline src/main/java/eu/execom/labs/teamodoro/service/UserService.java]: Why does this need to be transactional?

Comment (dufajlusi) [inline src/main/java/eu/execom/labs/teamodoro/service/UserService.java]: This is maybe my bad habit in this case when using find(get). @Transactional is not strictly necessary with ORM especially because set propagation mode to required by default, transaction is started when in fact one is not required. I am put just in case, because we don't know in witch direction will develop app...  **Will be removed**

Comment (stanislavgrujic) [inline src/main/java/eu/execom/labs/teamodoro/service/UserService.java]: As a general practice - you don't add something unless you're sure you're gonna need it. Having find method as transactional could indicate that there is a side effect in the method, which is a bad thing.

File: src/main/webapp/WEB-INF/views/jsp/layout/DashboardFooter.jsp [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  </div>
+
+  <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+  <script src="http://code.jquery.com/jquery-1.12.0.min.js"></script>
+  <!-- Include all compiled plugins (below), or include individual files as needed -->
+  <script src="http://netdna.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+  <script src="/resources/core/js/pomodoro-timer.js"></script>
+
+</body>
+</html>
```
Comment (stanislavgrujic) [inline src/main/webapp/WEB-INF/views/jsp/layout/DashboardFooter.jsp]: Please fix formatting in the file. Where are the starting elements of <body> and <html>? They should not be split in different files, it can be confusing, and does not help readability.

File: src/main/webapp/WEB-INF/views/jsp/layout/DashboardFooter.jsp [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        <p>&copy; execom.eu 2016</p>
+      </footer>
+    </div>
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="http://code.jquery.com/jquery-1.12.0.min.js"></script>
+    <!-- Include all compiled plugins (below), or include individual files as needed -->
+    <script src="http://netdna.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+    <script src="/resources/core/js/pomodoro-timer.js"></script>
+  </body>
+</html>
```
Comment (stanislavgrujic) [inline src/main/webapp/WEB-INF/views/jsp/layout/DashboardFooter.jsp]: I strongly oppose to have *<body>* and *<html>* start and end tags split into different files. There are much better ways to make this part reusable. You could create a JSTL tag, for example that will add a header or a footer.

File: src/main/webapp/WEB-INF/views/jsp/layout/LoginHeader.jsp [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+          </a>
+          <ul class="dropdown-menu">
+            <li><a href="?lang=en"><span class="glyphicon glyphicon-flag"></span> English</a></li>
+            <li><a href="?lang=de"><span class="glyphicon glyphicon-flag"></span> Deutchland</a></li>
+          </ul>
+        </li>
+      </ul>
+    </div>
+  </div>
+</nav>
```
Comment (stanislavgrujic) [inline src/main/webapp/WEB-INF/views/jsp/layout/LoginHeader.jsp]: There's much duplication between LoginHeader and Dashboard header, as was as in LoginFooter and DashboardFooter. Could it be resolved somehow, so that the code is reused?

File: src/main/webapp/WEB-INF/views/jsp/layout/DashboardHead.jsp [hunk 5]
```diff
@@ -12,4 +12,5 @@
   <link rel="stylesheet" href="http://netdna.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <!-- Css style for timer -->
   <link href="/resources/core/css/pomodoro-timer.css" rel="stylesheet" />
+  <script src="/resources/core/js/pomodoro-timer.js"></script>
```
Comment (stanislavgrujic) [inline src/main/webapp/WEB-INF/views/jsp/layout/DashboardHead.jsp]: Why is this moved here from the footer?
